python/pyproject.toml

```diff
@@ -27,6 +27,7 @@ runtime_common = [
     "llguidance>=0.7.11,<0.8.0",
     "modelscope",
     "msgspec",
+    "netifaces",
```

Comment:
Looks like this package is archived https://github.com/al45tair/netifaces, thus wondering whether it is good we make it a default dependency. One way may be make it optional dependency, and tell users to install it only when it is imported and must be used